imports/fix.go

```diff
@@ -181,7 +181,12 @@ func collectReferences(refs map[string]map[string]bool) visitFn {
 	return visitor
 }
 
-func fixImports(fset *token.FileSet, f *ast.File, filename string) (added []string, err error) {
+// FixImports adjusts the imports in the AST and returns a list of the imports
+// that have been added. The returned list of added imports is always the full
+// quoted import path of the package, not including the package name even if
+// the package name is inserted into the AST (eg. "test.v1" is returned but
+// 'test "test.v1" is inserted in the AST).
+func FixImports(fset *token.FileSet, f *ast.File, filename string) (added []string, err error) {
```

Comment:
warning: cyclomatic complexity 36 of function FixImports() is high (> 15) (gocyclo)